app.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const { getCategories } = require("./controllers");
+const app = express();
+
+app.get("/api/categories", getCategories);
+
+app.use((err, req, res, next) => {
```

Comment:
You've done error handling without any testing. For this categories ticket - we don't actually need anything other than the 500, but please remove the other error handling until the time comes when you need it, and ensure it's built up using TDD.